app/components/Documentation/helpers.tsx

```diff
@@ -180,3 +182,21 @@ export const isEditableChildItem = (
   }
   return false;
 };
+
+export const getDataFromTree = (
+  documentationMenu: Array<DocumentationItemMenuType>,
+) => {
+  return getFlatDataFromTree({
+    treeData: documentationMenu?.map((node) => ({ ...node })),
```

Comment:
For what this?

Reply:
This is the code from the official documentation of the library